src/ranged.cpp

```diff
@@ -189,24 +201,16 @@ bool player::handle_gun_damage( item &it )
         // Here we check for a chance for the weapon to suffer a mechanical malfunction.
         // Note that some weapons never jam up 'NEVER_JAMS' and thus are immune to this
         // effect as current guns have a durability between 5 and 9 this results in
-        // a chance of mechanical failure between 1/64 and 1/1024 on any given shot.
-        // the malfunction may cause damage, but never enough to push the weapon beyond 'shattered'
-    } else if( ( one_in( 2 << effective_durability ) ) && !it.has_flag( "NEVER_JAMS" ) ) {
+        // a chance of mechanical failure between 1/(64*3) and 1/(1024*3) on any given shot.
+        // the malfunction can't cause damage
+    } else if( ( one_in( ( 2 << effective_durability ) * 3 ) ) && !it.has_flag( "NEVER_JAMS" ) ) {
```

Comment:
Sounds like mechanical failure rate would double once the gun starts to have the slightest amount of dirt or blackpowder fouling.

Reply:
You are correct; adjusted.